src/GuppyDataExplorer/ExplorerSurvivalAnalysis/index.jsx

```diff
@@ -43,47 +47,91 @@ function ExplorerSurvivalAnalysis() {
-                endTime={endTime}
-                startTime={startTime}
-                timeInterval={timeInterval}
-              />
-            )}
-            {config.result?.risktable && (
-              <RiskTable
-                data={risktable}
-                endTime={endTime}
-                startTime={startTime}
-                timeInterval={timeInterval}
-              />
+          <div className='explorer-survival-analysis__column-right'>
+            {/* eslint-disable-next-line no-nested-ternary */}
+            {isUpdating ? (
+              <Spinner />
+            ) : isError ? (
+              <div className='explorer-survival-analysis__error'>
+                <h1>Error obtaining survival analysis result...</h1>
+                <p>
+                  Please retry by clicking {'"Apply"'} button or refreshing the
+                  page. If the problem persists, please contact administrator
+                  for more information.
+                </p>
+              </div>
+            ) : (
+              <>
+                {config.result?.survival && (
+                  <SurvivalPlot
+                    data={survival}
+                    endTime={endTime}
+                    startTime={startTime}
+                    timeInterval={timeInterval}
+                  />
+                )}
+                {config.result?.risktable && (
+                  <RiskTable
+                    data={risktable}
+                    endTime={endTime}
+                    startTime={startTime}
+                    timeInterval={timeInterval}
+                  />
+                )}
+              </>
             )}
-          </>
-        )}
-      </div>
+          </div>
+        </>
+      ) : (
+        <div className='explorer-survival-analysis__user-agreement'>
+          <p>
+            In order to use the Survival Analysis analytics tool, you must agree
+            to the following:
+          </p>
+          <ul>
+            <li>
+              I will not engage in “p-hacking” or other forms of statistical
+              abuse.
+            </li>
+            <li>
+              I will maintain a hypothesis record and declare a hypothesis
+              before analyzing data.
+            </li>
+            <li>
+              I will not reproduce or distribute results generated using
+              analytics tools.
+            </li>
+            <li>
+              I understand and agree that use of analytics tools is logged and
+              monitored to identify abuse and improve the tools. Individual user
+              data may be shared with partners of the PCDC (e.g., member
+              consortia) for operational purposes. Aggregate or de-identified
+              user data may be shared outside of the PCDC without limitation.
+            </li>
+            <li>
+              I understand that access may be revoked if a user is suspected to
+              engage in statistical abuse.
```

Comment:
I forgot if we talked about this. Not sure if this will change, I assume not frequently but it may be worth pulling from gitops.json. Did Brian or Kirk say it will stay the pretty much the same?

Reply:
I'm not so sure how often the verbiage will change at this point. If we want to make it a config option, I can make it work in a follow-up PR. We can quickly discuss it during today's team meeting.

Reply:
Update: I brought this up during today's meeting and the response from Brian is that we don't expect it to change often and so hard-coded text is acceptable. With that, I'm marking this as resolved.